content/manuals/desktop/features/gpu.md

```diff
@@ -63,16 +63,28 @@ GPU Device 0: "GeForce RTX 2060 with Max-Q Design" with compute capability 7.5
 = 2724.379 single-precision GFLOP/s at 20 flops per interaction
 ```
 
-## Run a real-world model: Llama2 with Ollama
+## Run a real-world model: smollm2 with Docker Model Runner
```

Comment:
We should also make this clear that this is for Docker Desktop 4.54 and later

Reply:
WDYT of a note?
<img width="909" height="334" alt="Screenshot 2025-12-05 at 11 46 36" src="https://github.com/user-attachments/assets/dd662624-bb19-4668-b9a7-7ef4e82d15e3" />

Reply:
yes perfect @doringeman, thank you